app/javascript/modules/listingDetailsAside/ListingDetailsSeeTheUnit.tsx

```diff
@@ -41,15 +75,7 @@ export const ListingDetailsSeeTheUnit = ({ listing }: SeeTheUnitProps) => {
           <ListingDetailsOpenHouses listing={listing} sectionHeader={false} />
         ) : null}
       </SeeTheUnitSubsection>
-      {listing.Multiple_Listing_Service_URL && (
-        <SeeTheUnitSubsection title={t("seeTheUnit.seeDetailsOnline")}>
-          <p>
-            <a href={listing.Multiple_Listing_Service_URL} target="_blank">
-              {t("listings.process.seeTheUnitOnMls")}
-            </a>
-          </p>
-        </SeeTheUnitSubsection>
-      )}
+      {SeeDetailsOnline(listing)}
```

Comment:
I think the convention in the codebase so far for a function (rather than a component) is with the first letter lowercase, so seeDetailsOnline

Reply:
I think we have a combination of both. Im seeing in this [file](https://github.com/SFDigitalServices/sf-dahlia-web/blob/44899ae12c3cd037609733fbebc53ca8530507c9/app/javascript/modules/listingDetailsAside/ListingDetailsSeeTheUnit.tsx#L21) and others the function components starting with an uppercase.